test/parallel/test-fs-realpath.js

```diff
@@ -201,9 +201,11 @@ function test_cyclic_link_protection(callback) {
     fs.symlinkSync(t[1], t[0], 'dir');
     unlink.push(t[0]);
   });
-  assert.throws(function() { fs.realpathSync(entry); });
+  assert.throws(function() { fs.realpathSync(entry); }, /^Error: ELOOP/);
```

Comment:
You can use `common.expectError(undefined, Error, 'ELOOP`)` here I believe. (Or some variation on that)